cerbero/packages/wix.py

```diff
@@ -71,21 +71,16 @@ def __init__(self, config, package):
         # wix_wine_prefix is a symbolic link to the regular cerbero prefix
         # and should not be a folder.
         if self._with_wine:
-          self.wix_wine_prefix = '/tmp/flu_cwwp'
-          if os.path.exists(self.wix_wine_prefix):
-            os.remove(self.wix_wine_prefix)
-          os.symlink(config.prefix, self.wix_wine_prefix)
-          self.prefix = self.wix_wine_prefix
+          if os.path.exists(get_wix_wine_prefix()):
+            os.remove(get_wix_wine_prefix())
+          os.symlink(config.prefix, get_wix_wine_prefix())
```

Comment:
In Windows os.symlink creates a full copy of the prefix

Reply:
IIRC you told me this is a hack for when the paths are too long, and this issue will happen also in a native windows when the prefix root is too long.
What is exactly the problem with long paths? Is it  with the ID's in the msis packaging step?
If that's the issue than the fix must be handled differently because we generate those id's based on the path, so we could compute those id's on a different way.

Reply:
On windows we dont have the possibility to create symlink but i guess using Z:\ as the build directory path, is hiding the bug, windows 10 I think does not own this bug anymore.
Its not a question of IDs but of resources path.

Reply:
And what is exactly the error? I would like to read to understand the real issue

Reply:
The error is related to the len of the file path. On linux it can /opt/build_blablab or even as my setup is right now in cerbero folder, ie $HOME/DEV/INTERNAL/fluendo-cerbero/build_cerbero. This path can exceed in old limitation of 128 chars or 256 depending on the OS or .NET 
https://github.com/wixtoolset/issues/issues/5314